changelog.d/10843.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug causing the `remove_stale_pushers` background job to repeatedly fail and log errors, when SQLite is in use and Synapse is upgraded from version 1.28 or older.
```

Comment:
nit: the comma here doesn't sound right

Reply:
you're right; was trying to make it more digestible but didn't work.